services/QuillLMS/db/migrate/20230413140558_create_canvas_accounts.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCanvasAccounts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :canvas_accounts do |t|
```

Comment:
Should this be a habtm between users and configs rather than some relationship between schools/districts and configs? How do schools/districts fit into this current structure? I also wonder if we should lean on the SchoolAdmin/DistrictAdmin relationship for permission-ing access to users here.

It might be worth us talking through / mapping out the modeling of this feature and the rest of Canvas.